WorkloadManagementSystem/DB/JobLoggingDB.py

```diff
@@ -57,14 +55,14 @@ def addLoggingRecord( self,
       time_order = round( epoc, 3 )
     else:
       try:
-        if type( date ) in StringTypes:
+        if isinstance( date, basestring ):
           # The date is provided as a string in UTC
           _date = Time.fromString( date )
           epoc = time.mktime( _date.timetuple() ) + _date.microsecond / 1000000. - MAGIC_EPOC_NUMBER
           time_order = round( epoc, 3 )
-        elif type( date ) == Time._dateTimeType:
+        elif isinstance( date, Time._dateTimeType ):
```

Comment:
datetime.datetime ? 
Maybe we can slowly look at the interest of keeping such module.